engines/osxnative/engine.go

```diff
@@ -0,0 +1,52 @@
+//go:generate go-composite-schema --unexported --required config config-schema.yml generated_configschema.go
+//go:generate go-composite-schema --unexported --required payload payload-schema.yml generated_payloadschema.go
+
+// Package macengine implements the Mac OSX engine
+package macengine
```

Comment:
package name should match the directory name usually.  I would opt to call this "osxnative"